pipeline/inferrer/inference_manager/src/test/scala/weco/pipeline/inference_manager/adapters/FeatureVectorInferrerAdapterTest.scala

```diff
@@ -43,15 +43,16 @@ class FeatureVectorInferrerAdapterTest
     it("augments InferredData with the data from the inferrer response") {
       val features = (0 until 4096).map(_ / 4096f).toList
       val featuresB64 = Encoding.toLittleEndianBase64(features)
-      val lshEncodedFeatures = ('a' to 'z').map(_.toString * 3).toList
+      val reducedFeatures = (0 until 1024).map(_ / 1024f).toList
+      val reducedFeaturesB64 = Encoding.toLittleEndianBase64(reducedFeatures)
```

Comment:
This makes me a bit uneasy, because we're doing both rounds of the base64-ing in Scala. I don't actually know what a feature inferrer response looks like; if we could save a real response into a JSON file and load it here, I'd find it a bit more useful.

Not blocking, but a thought.